services/py-api/src/utils.py

```diff
@@ -43,3 +45,37 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+class JwtUtility:
+    class UserData(TypedDict):
+        id: int
+        name: str
+        email: str
+        is_admin: bool
+        email_verified: bool
+        team_id: int
+        created_at: str
+        updated_at: str
```

Comment:
Why is there class nesting? This is not right. We already have a model for both the Participant and Team, including schemas for requests and responses. So you don't need to create your own model. Navigate to 

```Python
cd /services/py-api/database/model/participant_model.py
```

The participant adds the participant specific properties on top of the base model. So you should have all the properties in UserData available in the participant_model. Just import it. 

```python
from src.database.model.participant_model import Participant
```